portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,10 +35,99 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+   // private List<String> messages;
+
+  //  private List<String> comments = new ArrayList<>();
+
+
+  @Override
+  public void init(){
+     /* messages = new ArrayList<>();
+      messages.add("hardcode list 1");
+      messages.add("hardcode list 2");
+      messages.add("hardcode list 3");
+      */
+  }  
+  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("we got the data servlet connected to the index.html and script.js");
+    //response.setContentType("text/html;");
+   //response.getWriter().println("we got the data servlet connected to the index.html and script.js");
     //response.getWriter().println("<h1>Hello Joy K!</h1>");
+    //response.getWriter().println(messages);
+/*
+ response.setContentType("application/json;");
+
+      //Convert the server stats to JSON  
+String json = convertToJsonUsingGson(comments);
+
+    //Send the JSON as the response
+response.getWriter().println(json);
+*/
+
+    Query query = new Query("Comment");
+    //.addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> comments = new ArrayList<>();
+    System.out.println(comments);
+    for (Entity c : results.asIterable()) {
+        String name = (String) c.getProperty("name");
+        String comment = (String) c.getProperty("text-input");
+        
+    String toReturn = name + ": " + comment;
+      comments.add(toReturn);
+
+    }
+
+     response.setContentType("application/json;");
+     String json = convertToJsonUsingGson(comments);
+     response.getWriter().println(json);
+
+  
+   }
+
+   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the input from the form.
+
+     /* comments.add(eachcomment);
+    response.setContentType("application/json;");
+    String json = convertToJsonUsingGson(comments);
+    response.getWriter().println(json);
+    */
+    
+    String eachcomment = getParameter(request, "text-input", "");
```

Comment:
This is a personal preference thing, but I would probably call this `comment` instead of `eachcomment`. When I first read `eachcomment` I thought this was going to be an array or a list of multiple comments, but it's just a single comment, right?